Sevenpros/Teamwork
# Build Endpoint for Creating articles using database

#### What does this PR do?
Having endpoint for sharing articles using database working
#### Description of the task to be completed
* create article controller file
* create article model file
* write functions to manipulate articles
* write business logic inside article controller
* add route for sharing article
#### How can this be manually tested?
After cloning the repo in the local system change directory to the downloaded folder and start the app.
You are now ready to  use postman to check if endpoint is working as expected
#### Below are the screenshot showing how user successfully posted article, what happens in case of invalid article and without a token respectively.
![image](https://user-images.githubusercontent.com/26649756/66450294-4066a080-ea58-11e9-8cd8-96a5d16002dc.png)
![image](https://user-images.githubusercontent.com/26649756/66450301-48bedb80-ea58-11e9-904e-7684423c36aa.png)
![image](https://user-images.githubusercontent.com/26649756/66450357-7dcb2e00-ea58-11e9-82e9-df814fee3e0c.png)
 
#### What are the Relevant pivotal story
[#169029075](https://www.pivotaltracker.com/story/show/169029075)
